TTS/tts/datasets/formatters.py

```diff
@@ -59,46 +59,56 @@ def mozilla_de(root_path, meta_file):
     return items
 
 
-def mailabs(root_path, meta_files=None):
+def mailabs(root_path, meta_files=None, ununsed_speakers=None):
     """Normalizes M-AI-Labs meta data files to TTS format
-    
+
     Args:
-        root_path (str): root folder of the MAILAB language folder. 
+        root_path (str): root folder of the MAILAB language folder.
         meta_files (str):  list of meta files to be used in the training. If None, finds all the csv files
             recursively. Defaults to None
     """
     speaker_regex = re.compile("by_book/(male|female)/(?P<speaker_name>[^/]+)/")
-    if meta_files is None:
```

Comment:
This solves a bug ! In the dataset config the  meta_file_train and meta_file_val are defined as str so when we pass the metadata for this function  meta_files  will be empty str ("") and it will generate a Bug ...